qutip/cy/openmp/cqobjevo_omp.pyx

```diff
@@ -1,37 +1,5 @@
 #cython: language_level=3
-# distutils: language = c++
```

Comment:
Do we need to keep this or not?

Reply:
It has no effect, because we override it at the `setup.py` level, so I just left it as removed.